gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/TTimestampWrapper.java

```diff
@@ -1,16 +1,40 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 package com.farao_community.farao.cse_valid.app;
 
+import com.farao_community.farao.commons.EICode;
+import com.farao_community.farao.cse_valid.api.exception.CseValidInvalidDataException;
+import com.farao_community.farao.cse_valid.app.configuration.EicCodesConfiguration;
+import com.farao_community.farao.cse_valid.app.ttc_adjustment.TCalculationDirection;
 import com.farao_community.farao.cse_valid.app.ttc_adjustment.TTimestamp;
+import com.powsybl.iidm.network.Country;
 import xsd.etso_core_cmpts.QuantityType;
 
 import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * @author Vincent Bochet {@literal <vincent.bochet at rte-france.com>}
+ * @author Oualid Aloui {@literal <oualid.aloui at rte-france.com>}
+ */
 
 public class TTimestampWrapper {
     private final TTimestamp timestamp;
+    private final EicCodesConfiguration eicCodesConfiguration;
+    private Map<String, Boolean> exportingCountryMap;
```

Comment:
Be careful of the map's name because it is not consistent with the get method now.